ui/redux/selectors.js

```diff
@@ -3,19 +3,7 @@ import orderBy from 'lodash/orderBy'
 import uniqBy from 'lodash/uniqBy'
```

Comment:
everything removed here was moved either to ` ui/pages/Project/selectors.js` or `ui/shared/components/panel/variants/selectors.js`